hunts/models.py

```diff
@@ -47,27 +48,80 @@ def get_num_unlocked(self):
     def get_num_metas_solved(self):
         return self.puzzles.filter(Q(status=Puzzle.SOLVED), Q(is_meta=True)).count()
 
-    def get_solves_per_hour(self):
-        current_time = timezone.now()
+    def get_num_metas_unsolved(self):
+        return self.puzzles.filter(~Q(status=Puzzle.SOLVED), Q(is_meta=True)).count()
+
+    # Gets the number of puzzles that are either solved or feed into a solved meta.
+    def get_progression(self):
+        query = """
+        WITH RECURSIVE progression_puzzles (id) AS (
+            SELECT id
+            FROM puzzles_puzzle
+            WHERE (status = 'SOLVED' AND hunt_id = %s)
+            UNION
+            SELECT Metas.from_puzzle_id
+            FROM puzzles_puzzle_metas Metas
+            INNER JOIN progression_puzzles ON progression_puzzles.id = Metas.to_puzzle_id
+        )
+        SELECT id FROM progression_puzzles
```

Comment:
Does SELECT COUNT(id) ... not work? We only care about the number of IDs right? This is also more efficient.